src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -123,7 +123,7 @@ private async Task StartDashboardAsync(CancellationToken cancellationToken = def
             WorkingDirectory = dashboardWorkingDirectory
         };
 
-        if (StringComparer.OrdinalIgnoreCase.Equals(".dll", Path.GetExtension(fullyQualifiedDashboardPath)))
+        if (string.Equals(".dll", Path.GetExtension(fullyQualifiedDashboardPath), StringComparison.OrdinalIgnoreCase))
```

Comment:
Why this change? Seems equivalent but more verbose.

Reply:
I use them like this answer describes: https://stackoverflow.com/a/8918112